Functions/xASL_num2str.m

```diff
@@ -47,7 +52,7 @@
 
 % Default: set concatenate to 0
```

Comment:
Default goes to 1 also in the comment here

Reply:
9ebbd27